base/applications/mspaint/dialogs.h

```diff
@@ -43,10 +43,10 @@ class CAttributesDialog : public CDialogImpl<CAttributesDialog>
         COMMAND_ID_HANDLER(IDCANCEL, OnCancel)
         COMMAND_ID_HANDLER(IDD_ATTRIBUTESSTANDARD, OnDefault)
         COMMAND_ID_HANDLER(IDD_ATTRIBUTESRB1, OnRadioButton1)
-        COMMAND_ID_HANDLER(IDD_ATTRIBUTESRB1, OnRadioButton2)
-        COMMAND_ID_HANDLER(IDD_ATTRIBUTESRB1, OnRadioButton3)
+        COMMAND_ID_HANDLER(IDD_ATTRIBUTESRB2, OnRadioButton2)
+        COMMAND_ID_HANDLER(IDD_ATTRIBUTESRB3, OnRadioButton3)
         COMMAND_ID_HANDLER(IDD_ATTRIBUTESEDIT1, OnEdit1)
-        COMMAND_ID_HANDLER(IDD_ATTRIBUTESEDIT1, OnEdit2)
+        COMMAND_ID_HANDLER(IDD_ATTRIBUTESEDIT2, OnEdit2)
```

Comment:
When I see this code and read "OnRadioButtonX" I know immediately which action it is doing: switching the color space from RGB to black/white! ...
...
... ah no? actually it doesn't.

x-(